CondFormats/PCLConfig/plugins/AlignPCLThresholdsWriter.cc

```diff
@@ -0,0 +1,245 @@
+// -*- C++ -*-
+//
+// Package:    CondFormats/PCLConfig
+// Class:      AlignPCLThresholdsWriter
+// 
+/**\class AlignPCLThresholdsWriter AlignPCLThresholdsWriter.cc CondFormats/PCLConfig/plugins/AlignPCLThresholdsWriter.cc
+
+ Description: class to build the SiPixelAli PCL thresholds
+
+*/
+//
+// Original Author:  Marco Musich
+//         Created:  Wed, 22 Feb 2017 12:04:36 GMT
+//
+//
+
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "CondFormats/PCLConfig/interface/AlignPCLThresholds.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "CondCore/DBOutputService/interface/PoolDBOutputService.h"
+
+//
+// class declaration
+//
+
+namespace DOFs {
+  enum dof{
+    X,
+    Y,
+    Z,
+    thetaX,
+    thetaY,
+    thetaZ,
+    extraDOF
+  };
+}
+
+class AlignPCLThresholdsWriter : public edm::one::EDAnalyzer<edm::one::SharedResources>  {
```

Comment:
@mmusich I see that you declare it with `edm::one::SharedResources`, but you don't use declare the usage of any resources.